lesson05/shape.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+)
+
+type Shape interface {
+	Area() (float64, error)
+	Perimeter() (float64, error)
+}
+
+func DescribeShape(s Shape) string {
```

Comment:
This function should not be changed. It must be the same, as it was in the previous homework. Also, since it doesn't return anything, it should not be tested - you can remove the subsequent test